client/src/styles/_forms.less

```diff
@@ -31,8 +31,9 @@
   p.form-control {
     border: none;
     padding-left: 0;
-    padding-top: 8px;
+    margin: 6px 0 0 0 !important;
```

Comment:
Why do we need `!important`?

Reply:
This is because of this https://github.com/camunda/camunda-modeler/blob/f46e7d1133542476aa1f7dea197a0845dcd8880b/client/src/shared/ui/Section.less#L34

I don't think we are making use of the `p` one at the moment, but can't be 100% sure

Reply:
In 5.0.0-alpha.0, we used padding to achieve the same result 🤡  I think we shouldn't introduce `!important` to override styles we define in our own app. It's just a couple of places to check...

![image](https://user-images.githubusercontent.com/28307541/161587345-6510bca6-bc3d-45be-9a3b-000db9c03b79.png)

Reply:
Changed it